base/linalg/cholesky.jl

```diff
@@ -445,13 +445,13 @@ function getindex{T<:BlasFloat}(C::CholeskyPivoted{T}, d::Symbol)
     throw(KeyError(d))
 end
 
-show{T,S<:AbstractMatrix}(io::IO, C::Cholesky{T,S}) =
+show{T}(io::IO, C::Cholesky{T,<:AbstractMatrix}) =
```

Comment:
Giving a special meaning to underscore like that is not at all intuitive. Saving 4 characters isn't worth ending up with code that requires finding an obscure manual section to understand.